### PR TITLE
Remove all product/target specific code

### PR DESCRIPTION
**Pull request by andr2000**
Product/target specific code moved to meta-xt-prod-xxx,
only leave generic code here

### CODE REVIEW

File: classes/xt_rcar.bbclass [hunk 1]
```diff
@@ -0,0 +1,8 @@
+rcar_unpack_proprietary_graphics() {
```
Comment (aanisov) [inline classes/xt_rcar.bbclass]: Rename it to i.e. rcar_unpack_evaproprietary to correspondent to the packages content as well as follow Renesas'es style

File: recipes-dom0/dom0-image-weston/dom0-image-weston.bb [hunk 2]
```diff
@@ -4,6 +4,7 @@ LICENSE = "MIT"
 
 inherit build_yocto
 inherit xt_quirks
+inherit xt_rcar
```
Comment (aanisov) [inline recipes-dom0/dom0-image-weston/dom0-image-weston.bb]: xt_rcar -> rcar_evaproprietary?

**Comment by aanisov** [comment]
Software evaluation packages contain graphics and multimedia, so I would recommend to reword commit message of https://github.com/xen-troops/meta-xt-images/pull/4/commits/93889fe98bd5aa09905d54b047398e66ed394f11  as well as follow Renesas'es naming style in code changes

**Review by aanisov — APPROVED** [review]
for https://github.com/xen-troops/meta-xt-images/pull/4/commits/1960d38e2ebf015eabd76e1c9e179a762741fdcd

Reviewed-by: Andrii Anisov andrii_anisov@epam.com

File: classes/xt_rcar.bbclass (hunk 1 above)
Comment (lorc) [inline classes/xt_rcar.bbclass]: Wow! Should we have whole `.bbclass` for renesas quirks? Isn't `.inc` file not enough?

Comment (andr2000) [reply classes/xt_rcar.bbclass]: done

File: recipes-dom0/dom0-image-weston/dom0-image-weston.bb (hunk 2 above)
Comment (andr2000) [reply recipes-dom0/dom0-image-weston/dom0-image-weston.bb]: not sure, I will move it to .inc file and will contain all common quirks related to R-Car, not only unpacking

File: classes/xt_rcar.bbclass (hunk 1 above)
Comment (andr2000) [reply classes/xt_rcar.bbclass]: classes/xt_rcar.inc, according to the below this is correct place for include files:
https://www.yoctoproject.org/docs/1.6/bitbake-user-manual/bitbake-user-manual.html#locating-include-and-class-files

Comment (andr2000) [reply classes/xt_rcar.bbclass]: Correction: inc files are not visible in classes directory w/o altering BBPATH variable
so, instead I'll put it into inc/xt_rcar.inc

**Comment by andr2000** [comment]
with the changes above, are you guys ok to put RB?

**Comment by aanisov** [comment]
please also reword "Move R-Car proprietary **graphics** unpack into inc file"

**Pull request merged**